ssl/statem/statem.c

```diff
@@ -138,6 +139,9 @@ void ossl_statem_set_renegotiate(SSL_CONNECTION *s)
 
 void ossl_statem_send_fatal(SSL_CONNECTION *s, int al)
 {
+    /* Some public APIs may call internal functions that fatal error */
```

Comment:
This has to do with `SSL_(CTX_)get_cert_to_compress()`, in order to avoid copying the code that generates the certificate message (which could ossify), it disables the fatal errors.